settings.php

```diff
@@ -978,6 +985,14 @@ function convertseconds($argument)
                                                                       ?>>Fahrenheit</label>
                                                     </div>
                                                 </div>
+                                                <h4>Administrator Email Address</h4>
+                                                <div class="form-group">
+                                                    <div class="input-group">
+                                                        <input type="text" class="form-control" name="adminemail"
+                                                               value="<?php echo htmlspecialchars($adminemail); ?>">
+                                                    </div>
+                                                </div>
+                                                </div>
```

Comment:
The third </div> here isn't needed, and causes display issues across all tabs of the settings page. 
If you remove it, the settings page appearance is fine.

Reply:
Removed and pushed the update!